src/main/java/bio/terra/service/filedata/flight/ingest/IngestBuildAndWriteScratchLoadFileAzureStep.java

```diff
@@ -88,4 +89,9 @@ void writeCloudFile(FlightContext flightContext, String path, Stream<String> lin
                 userRequest.getEmail()));
     azureBlobStorePdao.writeBlobLines(signedPath, lines);
   }
+
+  @Override
+  public StepResult undoOptionalStep(FlightContext context) {
```

Comment:
Is this necessary? I thought the default for optional step was success?

Reply:
It totally does and this is not necessary.